README.md

```diff
@@ -160,9 +137,133 @@ $ rover
 
 You can specify the working directory (where your configuration is living) and the Terraform binary location using flags.
 
-```
+```shell
 $ rover -workingDir "example/eks-cluster" -tfPath "/Users/dos/terraform"
 ```
 
 Once Rover runs on `0.0.0.0:9000`, navigate to it to find the visualization!
 
+
+## Container image
+
+Everything is dockerized and handled by [buildx bake](docker-bake.hcl) for an agnostic usage of this repo:
+
+There are two varient of image
+
+1. Simple: typically fat image
+
+- e.g., `docker run --rm -it -p 9000:9000 -v $(pwd):/src im2nguyen/rover`
+
+2. Slim: rover and terraform compressed by [UPX](https://github.com/upx/upx)
+
+  > Slim images will take little more time to build as terraform and rover both pass through upx compression
+
+- e.g., `docker run --rm -it -p 9000:9000 -v $(pwd):/src im2nguyen/rover:slim`
+
+> Create docker buildx builder when first time using
+> ```docker buildx create --use```
+
+```shell
+git clone --depth 1 https://github.com/im2nguyen/rover.git rover
+cd rover
+
+## Create local image
+docker buildx bake
```

Comment:
I received the following error while building. Can you look into it? Thank you!

```
$ docker buildx bake
[+] Building 0.0s (0/0)                                                                                                                                       
docker-bake.hcl:27
--------------------
  25 |     variable "TAGS" {
  26 |       default = [
  27 | >>>     "${REPO}:latest",
  28 |         "${REPO}:${VERSION}",
  29 |         "${REPO}:${VERSION}-${GIT_SHA}"
--------------------
error: docker-bake.hcl:27,8-12: Variables not allowed; Variables may not be used here., and 14 other diagnostic(s)
```

Reply:
these are my current docker specs. might be good to include this as prereqs for folks that are interested in building this locally

```
$ docker version
Client:
 Cloud integration: 1.0.14
 Version:           20.10.6
 API version:       1.41
 Go version:        go1.16.3
 Git commit:        370c289
 Built:             Fri Apr  9 22:46:57 2021
 OS/Arch:           darwin/amd64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.6
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.13.15
  Git commit:       8728dd2
  Built:            Fri Apr  9 22:44:56 2021
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.4.4
  GitCommit:        05f951a3781f4f2c1911b05e61c160e9c30eaa8e
 runc:
  Version:          1.0.0-rc93
  GitCommit:        12644e614e25b05da6fd08a38ffa0cfe1903fdec
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

Reply:
can you `docker buildx version`?
mine is 
```github.com/docker/buildx v0.7.1-docker 05846896d149da05f3d6fd1e7770da187b52a247```

Reply:
I think this is due to docker version. Mine is,
```log
$ docker version        
Client:
 Version:           20.10.12
 API version:       1.41
 Go version:        go1.17.5
 Git commit:        e91ed5707e
 Built:             Mon Dec 13 22:31:40 2021
 OS/Arch:           linux/amd64
 Context:           builders
 Experimental:      true

Server:
 Engine:
  Version:          20.10.12
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.17.5
  Git commit:       459d0dfbbb
  Built:            Mon Dec 13 22:30:43 2021
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          v1.5.8
  GitCommit:        1e5ef943eb76627a6d3b6de8cd1ef6537f393a71.m
 runc:
  Version:          1.0.3
  GitCommit:        v1.0.3-0-gf46b6ba2
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

Can you try upgrading your docker version if that won't break any of your existing projects.